docs/guides/expectations/creating_custom_expectations/how-to-create-custom-parameterized-expectations.md

```diff
@@ -2,4 +2,45 @@
 title: How to create custom parameterized Expectations
 ---
 
-This article is a stub.
+This guide will walk you through the process of creating Parameterized Expectations - very quickly. This method is only available using the new Modular Expectations API in 0.13.
+
+:::note Prerequisites 
+This how-to guide assumes you have already:
+
+[Set up a working deployment of Great Expectations](/docs/tutorials/getting-started/intro)
+:::
+
+A Parameterized Expectation is a capability unlocked by Modular Expectations. Now that Expectations are structured in class form, it is easier than every before to inherit fromthese classes and build similar Expectations that are adapted to your own needs.
+
+### Steps
+
+#### 1. Select an Expectation to inherit from
+
+For the purpose of this excercise, we will implement the Expectation “expect_column_mean_to_be_positive” - a realistic Expectation of the data that can easily inherit from expect_column_mean_to_be_between.
```

Comment:
All expectations should be code blocks to signify connection to core GE library.